### ISSUE TITLE
PDF icon not loadin

### ISSUE THREAD

**Issue by medev21** [FortAwesome/react-fontawesome#202]
Does react-fontaseome support PDF icon?  It's not loading on my end, other icons such as Github and Linkedin are displaying.

`<FontAwesome name='file-pdf' size="3x"/>`

is that the right format `file-pdf`?  I tried `filePDF` but still not loading...

**Comment by jacobsowles** [comment]
I'm having the same issue, though I can get the icon to load when I import and pass the faFilePdf object, like this:

```
import { faFilePdf } from '@fortawesome/free-regular-svg-icons';
library.add(faFilePdf);

<FontAwesomeIcon icon={faFilePdf} />
```

**Comment by jacobsowles** [comment]
It looks like if you pass in a string for the icon, the component assumes the icon lives in the fas (solid) package:

```
function normalizeIconArgs(icon) {
  // ...
  if (typeof icon === 'string') {
    return { prefix: 'fas', iconName: icon }
  }
}
```

So trying to use the regular file-pdf icon by passing the name as a string won't work.

**Comment by medev21** [comment]
So in this code:

```
import { faFilePdf } from '@fortawesome/free-regular-svg-icons';
library.add(faFilePdf);

<FontAwesomeIcon icon={faFilePdf} />
```

I'm getting an error ->` profilePage.jsx:43 Uncaught ReferenceError: library is not defined`

I added `FontAwesome.library...` but still getting error.  Is there something that needs to go before `library`?

**Comment by jacobsowles** [comment]
Ah, yes, I forgot to include that. You'll want to add this import:

`import { library } from '@fortawesome/fontawesome-svg-core';`

**Comment by medev21** [comment]
Ok, I think I need to add this:

`import { library, icon } from '@fortawesome/fontawesome-svg-core'`

However, icon is not displaying.  Name parameter is required??


```
import { library, icon } from '@fortawesome/fontawesome-svg-core'
import { faFilePdf } from '@fortawesome/free-regular-svg-icons';
library.add(faFilePdf);

<FontAwesome icon={faFilePdf} size="3x"/>
```

**Comment by medev21** [comment]
Oh I see, but how about f I wanted to add linkedin and github icons.  I don't see them in `fontawesomeicon`.   So I'm trying to build a social media list w/ pdf icon, linkedin, and github icons.  I kept `fontawesome` so I can use linkedin but it looks like it's interfering with `fontawesomeicon`

**Comment by jacobsowles** [comment]
Those social icons are in the `@fortawesome/free-brands-svg-icons` package. I updated the codesandbox above with the GitHub and Twitter icons.

**Comment by medev21** [comment]
yeah I just found out about it, but you got it before me. Thank you!

**Comment by medev21** [comment]
@jacobsowles Thanks so much for your help,  learned something new here!!
